src/com/example/powerup/Game.java

```diff
@@ -0,0 +1,108 @@
+package com.example.powerup;
+
+import com.example.powerup.R;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.ListIterator;
+
+import android.app.Activity;
+import android.content.pm.PackageInstaller.Session;
+import android.database.sqlite.SQLiteDatabase;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.EditText;
+import android.widget.ListView;
+import android.widget.TextView;
+import android.widget.Toast;
+
+public class Game extends Activity {
+
+	private DatabaseHandler mDbHandler;
+	private SQLiteDatabase mDb;
+	public List<Answer> Answers;
+	public Question Questions;
+	public Scenario Scene;
+	private ListView mainListView;
+	private TextView QuestionTextView;
+	private ArrayAdapter<String> listAdapter;
+
+	/** Called when the activity is first created. */
+	@Override
+	public void onCreate(Bundle savedInstanceState) {
+		super.onCreate(savedInstanceState);
+		setmDbHandler(new DatabaseHandler(this));
+		getmDbHandler().open();
+		try {
+			getmDbHandler().readCSVQuestion("Question.csv");
+			getmDbHandler().readCSVAnswer("Answer.csv");
+			getmDbHandler().readCSVScenario("Scenario.csv");
+		} catch (IOException e) {
+			e.printStackTrace();
+		}
+
+		UpdateQA();
+	}
+
+	private void UpdateQA() {
+		Answers = getmDbHandler().getAllAnswer(SessionHistory.CurrQID);
+		setContentView(R.layout.pgame);
+		// Find the ListView resource.
+		mainListView = (ListView) findViewById(R.id.mainListView);
+		listAdapter = new ArrayAdapter<String>(this, R.layout.simplerow,
```

Comment:
You can also do this in onCreate. Since you are calling recursively, put all setup calls in onCreate so they are only called once